sdk/src/client/bindings/python/iota_client/prefix_hex.py

```diff
@@ -0,0 +1,6 @@
+def utf8_to_hex(utf8_data):
+    return '0x'+utf8_data.encode('utf-8').hex()
+
+
```

Comment:
```suggestion
```

Reply:
My Python formatter is doing this :carlos:

Reply:
Yeah I saw later that there was a lot of double new line everywhere...

Reply:
Sorry, you can put it back

Reply:
Next time this gets formatted maybe :trollface:

Reply:
I wanted to add formatter for the bindings too at some point. It's annoying to let your formatter or even IDE do it and everyone does it differently 😆